docs/common/ca.md

```diff
@@ -6,109 +6,87 @@ The [certificate authorities](https://en.wikipedia.org/wiki/Certificate_authorit
 
-When installing software the CAs, we provide a few mechanisms in order to allow you to make informed decisions.
-You might ask "why do I care? Can’t you just give them to me?"  A few reasons why you may want to be involved:
+We provide three options for installing CA certificates that offer varying levels of control:
 
--   What set of CA certificates do you want? How much control do you want over the set of CA certificates?
-    Some sites might not want to install specific CAs for policy or security reasons.
--   How do you want to update them?
--   Do you want to centrally manage the CA certificates or install them on each host at your site?
-
-You have three options for installing CA certificates:
-
-1.  Install an RPM for a specific set of CA certificates.
-2.  Install `osg-update-certs`, a program that lets you install/update a predefined set of CA certificates, then adjust the set by adding or deleting specific CAs.
-3.  Install an RPM that installs **no** CAs. This is useful when you want your RPM installations to succeed (because our RPMs require CA certificates, and this RPM satisfies that dependency) but you want to manage them with your own technique.
-
-Additionally this page also provides instruction on installation of a tool (`fetch-crl`) to ensure your site has up-to-date certificate revocation list (CRL) from the CA.
+1.  Install an RPM for a specific set of CA certificates (*default*)
+2.  Install `osg-ca-scripts`, a set of scripts that provide fine-grained CA management
+3.  Install an RPM that doesn't install **any** CAs.
+    This is useful if you'd like to manage CAs yourself while satisfying RPM dependencies.
 
 Prior to following the instructions on this page, you must enable our [yum repositories](/common/yum.md)
 
 Installing CA Certificates
 --------------------------
 
-Please choose one of the three options to install the CA certificates.
+Please choose one of the three options to install CA certificates.
 
 ### Option 1: Install an RPM for a specific set of CA certificates ###
 
-If you want to install an RPM for one of our predefined CA certificates, you have two choices to make:
+!!! note
+    This option is the default if you install OSG software without pre-installing CAs.
+    For example, `yum install osg-ce` will bring in `osg-ca-certs` by default.
 
-#### Which set of CAs? ####
+In the OSG repositories, you will find two different sets of predefined CA certificates:
 
-1.  (*recommended*) The OSG CA certificates. This is similar to the IGTF set, but may have a small number of additions or deletions. (See [here](#contents-of-osg-ca-package) for details)
-2.  The default [IGTF](http://www.igtf.net/) CA certificates.
+- (*default*) The OSG CA certificates. This is similar to the IGTF set, but may have a small number of additions or deletions. (See [this section](#contents-of-osg-ca-package) for details)
```

Comment:
Maybe this is outside of the scope for this review, but a reason for why the admin should choose osg-ca-certs over igtf-ca-certs (or vice versa) might be nice. That might require contacting the security team though.

Reply:
I'll bring this up at the next security meeting